### PR TITLE
add login to ui-tests and fix responsiveness

### PR DESCRIPTION
**Pull request by harunurhan**
* Adds login for ui-tests

* Enables detail page tests with new screenshot and api recordings

* Fixes header responsiveness

### CODE REVIEW

File: src/common/components/DropdownMenu/DropdownMenu.jsx [hunk 1]
```diff
@@ -54,20 +49,25 @@ class DropdownMenu extends Component {
...
+        to: PropTypes.string,
+        display: PropTypes.string.isRequired,
+        target: PropTypes.string,
+      }),
+      PropTypes.node,
+    ])
   ).isRequired,
   title: PropTypes.node.isRequired,
   titleClassName: PropTypes.string,
+  dataTestId: PropTypes.string,
```
Comment (drjova) [inline src/common/components/DropdownMenu/DropdownMenu.jsx]: is this really necessary?

Comment (drjova) [reply src/common/components/DropdownMenu/DropdownMenu.jsx]: adding props just for the tests seems weird

Comment (harunurhan) [reply src/common/components/DropdownMenu/DropdownMenu.jsx]: not necessary if you're willing to have classname or some other kind of selector, which will result in mess in the future. 

Also I think adding a prop for testability is not a bad thing.

Comment (drjova) [reply src/common/components/DropdownMenu/DropdownMenu.jsx]: maybe it's better to rename it as a generic ``id`` prop instead of  a``test`` ``id``

**Pull request merged**